src/test/java/org/hl7/davinci/priorauth/ClaimResponseEndpointTest.java

```diff
@@ -178,8 +178,25 @@ public void getClaimResponseThatDoesNotExist() throws IOException {
 
     // Test that non-existent ClaimResponse returns 404.
     Request request = new Request.Builder()
-        .url(base + "/ClaimResponse?identifier=ClaimResponseThatDoesNotExist&patient.identifier=45").build();
```

Comment:
I am now a little puzzled as to why an empty bundle is returned with a 200, when I send this same request using postman I get a 404.

Reply:
I'm getting 200 with an empty bundle when I run this branch on postman

Reply:
From AEGIS: if you do a read on a resource and it's not there, it's a 404.   If you search for results and there are none, it's an empty bundle.

In this case it is a read not a search right? Because it has both an `identifier` and a `patient.identifier` I think it is classified as a read. So it should be a 404, no?

Reply:
Yes, this should be a 404 since it would be a read and not a search.